sonar-orchestrator/src/main/java/com/sonar/orchestrator/config/Licenses.java

```diff
@@ -44,30 +43,24 @@ public Licenses() {
     this("https://raw.githubusercontent.com/SonarSource/licenses/");
```

Comment:
Just a small remark: this project is public and we are defining the way of retrieving valid licences here.